### PR TITLE
[Docs] Fix build issue

### PR DESCRIPTION
**Pull request by gastonponti**
### Description

Docs build was failing due to dependencies issues

### CODE REVIEW

**Review by yorhodes — APPROVED** [review]
thanks for doing this! I know the work required was >>> what's visible

File: package.json [hunk 1]
```diff
@@ -91,7 +91,9 @@
     "randomatic": "^3.1.1",
     "logkitty": "^0.7.1",
     "minimist": "^1.2.5",
-    "npm": "^6.14.5",
+    "yargs-parser": "^13.1.2",
+    "npm": "^5.10.0",
```
Comment (jmrossy) [inline package.json:95]: We recently upgraded the npm version due to a security alert from github. Can you check if this reintroduces that?

Comment (gastonponti) [reply package.json:95]: Yes, I talked with @cmcewen (who made the security resolution), and he told me that it was ok.
I'm trying to see if there is a way to check this package from the github database, but I'm not finding any way

Comment (jmrossy) [reply package.json:95]: Cool thanks for checking

**Pull request merged**